arangod/Agency/Agent.cpp

```diff
@@ -1354,6 +1522,9 @@ void Agent::run() {
       // Check whether we can advance _commitIndex
       advanceCommitIndex();
 
+      // Clear expired long polls
+      clearExpiredPolls();
```

Comment:
Why is this needed twice? See line 1498.

Reply:
It is not